server/sonar-server/src/main/java/org/sonar/server/computation/queue/CeQueueInitializer.java

```diff
@@ -47,7 +49,14 @@ public CeQueueInitializer(DbClient dbClient, CEQueueStatus queueStatus, CeQueueC
   }
 
   @Override
-  public void start() {
+  public void onServerStart(Server server) {
+    if (!done) {
```

Comment:
For my information, why do you need to store this done state ? Can the onServerStart() be called many times on the same ServerStartHandler ?